CRM/Contact/Page/View.php

```diff
@@ -311,7 +311,13 @@ public static function setTitle($contactId, $isDeleted = FALSE) {
     }
     if ($isDeleted) {
       $title = "<del>{$title}</del>";
-      $mergedTo = civicrm_api3('Contact', 'getmergedto', ['contact_id' => $contactId, 'api.Contact.get' => ['return' => 'display_name']]);
+      try {
+        $mergedTo = civicrm_api3('Contact', 'getmergedto', ['contact_id' => $contactId, 'api.Contact.get' => ['return' => 'display_name']]);
+      }
+      catch (CiviCRM_API3_Exception $e) {
+        CRM_Core_Session::singleton()->setStatus(ts('Have not been able to determine which contact this contact merged into'), ts('Unable to determine kept contact'));
```

Comment:
@seamuslee001 so my last concern is the wording here - someone like @MikeyMJCO might have an idea but maybe something longer like 'This contact was deleted in a merge but the new contact can not be determined  / has since been deleted'

My main 'ug' is just the 'Have not been able' doesn't feel right  - who has not....